### ISSUE TITLE
[Bug] Week and Month fields don't work on Firefox and Safari

### ISSUE THREAD

**Issue by tabacitu** [Laravel-Backpack/CRUD#4474]
Those fields use the HTML5-spec input[type=week] and input[type=month], which Firefox and Safari do not currently support ([week](https://caniuse.com/mdn-html_elements_input_type_week), [month](https://caniuse.com/mdn-html_elements_input_type_month)).

We should either:
(a) warn people in their docs about that
(b) add a polyfill or something, if they exist
(c) instruct people to use a combination of `select_from_array` fields to achieve a similar result

**Comment by pxpm** [comment]
Hello @tabacitu 

I've been on the quest to close this issue. 

a) Warning about this in the docs is a must! (i will be doing it rigth away).
b) I've found some polyfills for it, but none of them seemd to be "properly working" or had lots of other dependencies with some behavioral changes.
c) I think the "workaround" should still be something around the "calendar" solution, so I'v come up with using the datepicker jquery to fix the `month` type problem:

```php
$this->crud->addField([   // Date
        'name'  => 'date_picker',
        'type'  => 'date_picker',
        'date_picker_options' => [
            'format'   => 'yyyy-mm',
            'language' => 'en',
            'minViewMode' => 'months'
        ],
    ]);
```
That looks something like this:
![image](https://user-images.githubusercontent.com/7188159/185104028-da3a7346-fc21-43d8-8021-e9f44685fe18.png)

For the week selection I have no other option, the only polyfill "working" that I found was https://github.com/aFarkas/webshim but it does not work with jQuery 3+. 

Even if we find some solution there would be always problems with mobile/touches etc etc that are very tricky to solve. 

My road plan here:
- document the limitations
- add "month" workaroud suggestion
- be done with this for now and hope they add support some day 🙃 

Maybe we can point developer in the direction that he can use select_from_array to achieve a "similar result selection" but I think it completely defeats the purpose of using week/month input types (for interface selection and input "regex constrain").

Let me know what you think @tabacitu 

Cheers
